content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management.rst

```diff
@@ -1,18 +1,144 @@
+that warehouse. Users can create and manage numerous locations within a single warehouse to organize
+inventory more precisely.
+
+.. seealso::
+   - :doc:`inventory_management/use_locations`
+   - :doc:`inventory_management/count_products`
+   - :doc:`inventory_management/cycle_counts`
+   - :doc:`inventory_management/scrap_inventory`
+
+.. _inventory/warehouses_storage/location-type:
+
+Location types
+--------------
+
+*Location types* in Odoo help categorize and manage where products are, and what actions need to be
+taken with them. By default, on the :menuselection:`Inventory app --> Configuration --> Locations`
+page, only internal locations are displayed.
+
+To view the seven location types in Odoo, select any location, and in the :guilabel:`Location Type`
+field, there are:
+
+- :guilabel:`Vendor Location`: defines an area where products purchased from vendors originate.
+  Items here are **not** in stock.
+
+- :guilabel:`View`: used to organize and structure the warehouse hierarchy. For example, the view
+  location `WH` (short for warehouse) groups all internal locations, such as `Stock`, receiving
+  docks, quality checkpoints, and packing areas to show they all belong to the same warehouse.
+
+  .. important::
+     View locations should **not** contain products, but it is possible to move them there.
+
+- :guilabel:`Internal Location`: storage locations within the warehouse. Items stored in these
+  locations are accounted for in :doc:`inventory valuation
+  <inventory_valuation/using_inventory_valuation>`.
+
+- :guilabel:`Customer Location`: where sold products are tracked; items here are no longer in stock.
+
+- :guilabel:`Inventory Loss`: counterpart location to consume missing items or create stock,
+  accounting for discrepancies.
+
+  In Odoo, examples of inventory loss locations are *Inventory Adjustment*, used to account for
+  discrepancies during an inventory count, and *Scrap*, which is where damaged goods are sent to
+  account for inventory losses.
+
+   .. example::
+      `Virtual Locations/Inventory Adjustment` is a location with the :guilabel:`Inventory Loss`
+      type. The database shows `65` units in `WH/Stock`, but an inventory check reveals `60`. To
+      correct the quantity, five units are moved from `WH/Stock` to `Virtual Locations/Inventory
+      Adjustment`.
+
+      .. image:: inventory_management/inventory_management/inventory-loss.png
+         :align: center
+         :alt: Product ends up in Virtual Locations/Inventory Adjustment.
+
+- :guilabel:`Production`: where raw materials are consumed, and :doc:`manufactured products
+  <../../manufacturing>` are created.
+
+- :guilabel:`Transit Location`: used for inter-company or inter-warehouse operations to track
+  products shipped between different addresses, such as :ref:`Physical Locations/Inter-warehouse
+  transit <inventory/warehouses_storage/interwarehouse-transit>`.
+
+.. image:: inventory_management/inventory_management/locations.png
```

Comment:
same here, this is hard to read
can we zoom in on the UI more here (or reduce the browser width a little bit, before mobile breakpoint)?

Reply:
got it, zoomed (: